build/gulp/tasks/test-projects/cra/App.tsx

```diff
@@ -25,7 +25,7 @@ class App extends React.Component {
             <Icon name="umbrella" circular bordered />
           </Animation>
           <Attachment header="Document.docx" />
-          <Avatar src="//placehold.it" />
+          <Avatar image={{ src: '//placehold.it' }} />
```

Comment:
```suggestion
          <Avatar image='//placehold.it' />
```

It seems that it can simplified...